scss/_patterns_navigation-reduced.scss

```diff
@@ -0,0 +1,120 @@
+@mixin vf-p-navigation-reduced {
+  @media (min-width: $breakpoint-navigation-threshold - 1) {
+    .p-navigation--sliding.is-reduced.has-search-open {
```

Comment:
In future the sliding option is probably going to be default. And I guess there is no need for "reduced" navigation that is not "sliding" at the same time.

So it feels that it would be quite natural for this to be `p-navigation--reduced` (the "main" modifier, because it is a significant difference in how the navigation renders).

Of course this will mean we have to make sure it extends everything it needs from `p-navigation--sliding` (or keep both class names on the element for the time being).